plans/fix-2052-windows-docker-esm-hook.md

```
@@ -0,0 +1,170 @@
+
+## User prompt
+
+> https://github.com/receptron/mulmoclaude/issues/2052 これ、つめていける？windowsのciなどを使って直したい。
+
+## What the reporter saw
+
+Windows 23H2 + Docker Desktop (WSL2), source checkout, sandbox ON, at commit `c15980d0`:
+
+1. Chatting → `ERROR [agent-stderr] Error: MCP tool mcp__mulmoclaude__handlePermission (passed via --permission-prompt-tool) not found.`
+2. `npx tsx --test test/agent/test_mcp_docker_smoke.ts` → **the exact same error text as before PR #1974**:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)